src/NuGet.Clients/NuGet.Console/Xamls/ConsoleContainer.xaml.cs

```diff
@@ -31,6 +30,7 @@ public ConsoleContainer(IVsShell4 shell)
                             var productUpdateService = ServiceLocator.GetInstance<IProductUpdateService>();
                             var packageRestoreManager = ServiceLocator.GetInstance<IPackageRestoreManager>();
                             var deleteOnRestartManager = ServiceLocator.GetInstance<IDeleteOnRestartManager>();
+                            var shell = ServiceLocator.GetGlobalService<SVsShell, IVsShell4>();
```

Comment:
This would just delay it right? 
Looking at the APIs it doesn't guarantee that it'll be initialized.